analyses/snv-callers/compare_snv_callers_plots.Rmd

```diff
@@ -278,20 +281,14 @@ perc_change_df <- all_caller_df %>%
       TRUE ~ base_change
     )
   ) %>%
-  # Whittle down to necessary columns
-  dplyr::select(change, dplyr::starts_with("VAF_")) %>% 
-  # Make this long form
-  tidyr::gather(key = "caller", value = "vaf", -change) %>% 
-  # Drop the prefix
-  dplyr::mutate(caller = gsub("VAF_", "", caller)) %>% 
-  # Get rid of NA rows
-  dplyr::filter(!is.na(vaf)) %>%
+  # Transmute VAFs to TRUE/FALSE
```

Comment:
Is this comment what is happening here? I am confused by this set of steps, but I think mostly because `all_caller_df` probably had a lot of extra columns that could be trimmed away before joining. You used to do that, but now you don't?

Reply:
This comment is out of date. I've selected to just `Allele` and `Reference_Allele` since that's all I really needed from there.